app/Services/Model/Imp/ProcessScheduleEntityLifeCycleEventService.php

```diff
@@ -222,6 +223,25 @@ public function process(string $entity_operator, int $summit_id, int $entity_id,
                 return;
             }
 
+            if ($entity_type === 'SummitTicketType') {
+                $summit = $this->summit_repository->getById($summit_id);
+                if (!$summit instanceof Summit) return;
+
+                $repository = EntityManager::getRepository(SummitTicketType::class);
```

Comment:
get the ticket type from summit to ensure the relationship

Reply:
done